style.css

```diff
@@ -288,6 +294,155 @@
     display: grid;
     grid-template-columns: auto;
     gap: 80px;
+    margin-top: 20px;
+  }
+
+  .modal {
+    margin-top: 50px;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%) scale(0);
+    transition: 200ms ease-in-out;
+    border: 1px solid black;
+    width: 90%;
+    height: auto;
+    background-color: #fff;
+    z-index: 10;
+  }
+
+  .modal.active {
+    transform: translate(-50%, -50%) scale(1);
+  }
+
+  .pop-up-header {
+    display: flex;
+    justify-content: space-between;
+    margin: 16px 30px 16px 16px;
+    align-items: center;
+  }
+
+  .popheading1 {
+    font-family: "Roboto", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    color: #172b4d;
+  }
+
+  .popheading2 {
+    display: none;
+  }
+
+  .close-button {
+    border: none;
+    font-size: 45px;
+    color: #67798e;
+    outline: none;
+    background: none;
+  }
+
+  .close-button:hover {
+    cursor: pointer;
+  }
+
+  .pop-up-lists {
+    display: flex;
+    justify-content: left;
+    list-style-type: none;
+    font-family: "Inter", sans-serif;
+    font-size: 13px;
+    font-weight: 600;
+    line-height: 16px;
+    text-align: left;
+    gap: 12px;
+    padding: 0;
+    margin-left: 16px;
+    margin-right: 15%;
+    margin-bottom: 24px;
+  }
+
+  .pop-list {
+    box-sizing: border-box;
+    border: 1px solid #8993a4;
+    align-items: center;
+    padding: 10px 12px;
+    background: #fff;
+    text-decoration: none;
+    color: #3c3a39;
+  }
+
+  .popcon > img,
+  .popuppharagraph1,
+  .modal-btns {
+    margin-left: 16px;
+    margin-right: 20px;
+  }
+
+  .popcon > img {
+    width: 85%;
+  }
+
+  .popuppharagraph1 {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    color: #344563;
+    margin-bottom: 16px;
+  }
+
+  .popuppharagraph2 {
+    display: none;
+  }
+
+  .pop-up-btns {
+    margin-right: 20px;
+    margin-left: 16px;
+    margin-bottom: 20px;
+    display: flex;
+    justify-content: space-between;
+    gap: 20px;
+  }
+
+  .pop-btn {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    width: 48%;
+    color: #fff;
+    background: #ff6b00;
+    border: none;
+    display: flex;
+    justify-content: center;
+    gap: 20px;
+    text-decoration: none;
+  }
+
+  .pop-btn:hover {
+    cursor: pointer;
+  }
+
+  #overlay {
+    position: fixed;
+    opacity: 0;
+    transition: 200ms ease-in-out;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: rgba(0, 0, 0, 0.8);
+    pointer-events: none;
+    filter: blur(15px);
+    -webkit-filter: blur(15px);
+  }
+
+  #overlay.active {
+    opacity: 1;
+    pointer-events: all;
   }
```

Comment:
Great job team 🤝

- [ ] [Optionla] Kindly fix the space below the mobile popup. This will make your design a replica of the design provided in the Figma template
![mobile-popup](https://user-images.githubusercontent.com/63917725/195522843-c17bfd85-cad6-48a4-b362-9c9e52fec4fc.PNG)